CMakeLists.txt

```diff
@@ -109,11 +109,36 @@ install(TARGETS minizip EXPORT minizip-exports
   LIBRARY DESTINATION "lib"
   ARCHIVE DESTINATION "lib")
 
+set(INSTALL_CMAKE_DIR "cmake")
 install(EXPORT minizip-exports
-        DESTINATION "cmake"
+        DESTINATION "${INSTALL_CMAKE_DIR}"
         NAMESPACE "MINIZIP::")
 
-install(FILES ${MINIZIP_PUBLIC_HEADERS}
+# Create and install CMake package config version file to allow find_package()
+include(CMakePackageConfigHelpers)
+write_basic_package_version_file(${CMAKE_CURRENT_BINARY_DIR}/minizipConfigVersion.cmake
```

Comment:
`minizip-config-version.cmake` would be more consistent with `minizip-exports.cmake`, but the master branch uses `minizipConfigVersion.cmake`.

Reply:
If there is no expected way to name things, I don't mind changing master branch.

Reply:
I have changed it in master `dev` branch.

Reply:
Thanks. Renamed to `minizip-config-version.cmake`.